base/exports.jl

```diff
@@ -1010,7 +1010,6 @@ export
     Condition,
     consume,
     current_task,
-    istaskstarted,
```

Comment:
In various intermediate versions of the PR, it wasn't possible, so I eliminated it. Later on, I didn't see an obvious need to reimplement it. Plus, the only usage of it in Base (in serialize) appears to be a race condition.